lcu_integration.py

```diff
@@ -17,43 +22,71 @@
 # We want to implement a synchronous approach,
 # so we are not using the library.
 def get_lcu_process():
+    """
+    Get the LeagueClientUx process, aka. the process of the League client.
+
+    Returns:
+        The process if found, else None.
+
+    """
     for process in process_iter():
-        if process.name() in {'LeagueClientUx.exe', 'LeagueClientUx'}:
```

Comment:
Hmm, so typically I've always leaned into single-quotes-unless-necessary (biased by other languages which have stricter styleguides in that regard), so had applied that on this project. It looks like [PEP-8 dictates](https://peps.python.org/pep-0008/) that you should pick one convention and stick to it, while [CKAN](https://docs.ckan.org/en/ckan-2.1.5/python-coding-standards.html) believes in single-quotes-unless-escaped.

I don't have crazy strong feelings about this, but if you don't either I think I'd like to stick to that ideology.

Reply:
Black made that change with the reasoning
> Why settle on double quotes? They anticipate apostrophes in English text. They match the docstring standard described in [PEP 257](https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring). An empty string in double quotes ("") is impossible to confuse with a one double-quote regardless of fonts and syntax highlighting used. On top of this, double quotes for strings are consistent with C which Python interacts a lot with.

I actually double checked and there ware two places where we had single quotes instead of double quotes, so I'd prefer to let black do its thing instead of making an exception for those lines. (My background is also in C and Java, so I'm a bit biased on strings 😅)

If you still prefer single-quote strings, there's a CLI flag to skip this step in black (`--skip-string-normalization`), but there's no "use single quotes" option, sadly.

Reply:
Ahh right, I remember black being a bit of a PITA because it's "config-less highly biased to its own set of rules", so if that's easier let's just stick with that 😛